### PR TITLE
add doc manage error fragment

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: en/developer_guide/error_pages.rst [hunk 1]
```diff
@@ -83,3 +83,29 @@ language and a Symfony 404 page for others.
 Note: like 503 pages, 404 pages are also dumped by the console command orchestra:errorpages:generate. They
 can so be used with virtual hosts configuration on some cases if needed.
 
+Error inside a block
+--------------------
+
+In Orchestra applications when a block throws an error or an exception the
```
Comment (ngilain) [inline en/developer_guide/error_pages.rst]: This sentence should be rephrased, it's a little hard to understand

File: en/developer_guide/error_pages.rst [hunk 2]
```diff
@@ -83,3 +83,29 @@ language and a Symfony 404 page for others.
...
+`default mechanical of Symfony is used <http://symfony.com/doc/current/controller/error_pages.html#overriding-the-default-error-templates>`_
+
+But to determine the template filename, the format used is fragment.<format>, for instance to render an error 500 in html Open Orchestra
```
Comment (ngilain) [inline en/developer_guide/error_pages.rst]: `<format>. For instance, to render`

**Comment by alavieille** [comment]
@ngilain pr updated

**Comment by ngilain** [comment]
:+1:

**Pull request merged**